### PR TITLE
Code and Json for edx.team.deleted event

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Trello Link:** _https://trello.com/c/bgEsibO7/113-teams-related-event-edxteamdeleted_

**Description:** _When a team is deleted, the server emits an edx.team.deleted event._

**Checks before merge:**

- [ ] Reviewed
- [ ] Unit Test Pass

### CODE REVIEW

File: openedx/features/caliper_tracking/transformers/team_transformers.py [hunk 1]
```diff
@@ -66,7 +67,9 @@ def edx_team_learner_added(current_event, caliper_event):
         ))
     )
 
-    object_link = utils.get_team_url_from_team_id(current_event['referer'], current_event['event']['team_id'])
+    object_link = utils.get_team_url_from_team_id(current_event['referer'],
+                                                  current_event['event'][
+                                                      'team_id'])
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/team_transformers.py]: Please refactor this into more appropriate form.

File: openedx/features/caliper_tracking/transformers/team_transformers.py [hunk 2]
```diff
@@ -112,3 +115,45 @@ def edx_team_learner_added(current_event, caliper_event):
     })
 
     return caliper_event
+
+
+def edx_team_deleted(current_event, caliper_event):
+    """
+    This event is generated when new team is created
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/team_transformers.py]: Please use the correct doc string.

File: openedx/features/caliper_tracking/transformers/team_transformers.py [hunk 3]
```diff
@@ -112,3 +115,45 @@ def edx_team_learner_added(current_event, caliper_event):
...
+
+    :param current_event: default event log generated.
+    :param caliper_event: caliper_event log having some basic attributes.
+    :return: updated caliper_event.
+    """
+
+    team_link = utils.get_team_url_from_team_id(current_event['referer'],
+                                                  current_event['event'][
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/team_transformers.py]: Please fix the indentation here.

File: openedx/features/caliper_tracking/transformers/team_transformers.py (hunk 1 above)
Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/transformers/team_transformers.py]: done.

File: openedx/features/caliper_tracking/transformers/team_transformers.py (hunk 2 above)
Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/transformers/team_transformers.py]: done.

File: openedx/features/caliper_tracking/transformers/team_transformers.py (hunk 3 above)
Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/transformers/team_transformers.py]: done.

**Pull request merged**